### PR TITLE
created and styled ChildInfo screen

### PR DESCRIPTION
**Pull request by crathor**

### CODE REVIEW

File: js/components/ChildInfoForm/ChildInfoForm.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
+import React, { Fragment } from 'react';
+import { TextInput, Text, View } from 'react-native';
+import { Form, Field } from 'react-final-form';
+import styles from './styles';
+import PropTypes from 'prop-types';
+
+const ChildInfoForm = ({ updateFormValues, child }) => {
+  console.log(child);
```
Comment (siddharthparmar7) [inline js/components/ChildInfoForm/ChildInfoForm.js]: please remove this console log

File: js/components/ChildInfoForm/ChildInfoForm.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  return (
+    <Form
+      onSubmit={() => {}}
+      validate={() => {}}
```
Comment (siddharthparmar7) [inline js/components/ChildInfoForm/ChildInfoForm.js]: Is this for later?

File: js/components/ChildInfoForm/ChildInfoForm.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            name="childName"
+            render={({ input, meta }) => (
+              <Fragment>
+                <Text style={styles.formTitle}>Child Name</Text>
+                <TextInput
+                  textContentType="name"
+                  placeholder="child name"
+                  {...input}
+                  value={child.childName}
+                  onChangeText={text => {
```
Comment (siddharthparmar7) [inline js/components/ChildInfoForm/ChildInfoForm.js]: text is not being used.

File: js/components/ChildInfoForm/ChildInfoForm.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+            name="parentName"
+            render={({ input, meta }) => (
+              <Fragment>
+                <Text style={styles.formTitle}>Parent Name</Text>
+                <TextInput
+                  textContentType="name"
+                  placeholder="parent name"
+                  {...input}
+                  value={child.parentName}
+                  onChangeText={text => {
```
Comment (siddharthparmar7) [inline js/components/ChildInfoForm/ChildInfoForm.js]: text is not being used.

File: js/screens/ChildInfo/ChildInfo.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+        <View style={styles.buttonContainer}>
+          <OutlinedButton
+            title={editChildInfo ? 'Done' : 'Edit'}
+            color={colors.teal}
+            pressed={
+              editChildInfo ? () => updateChildInfo() : () => showEditInfo()
+            }
+            titleStyle={{
+              marginTop: -5,
+              marginBottom: -7,
```
Comment (siddharthparmar7) [inline js/screens/ChildInfo/ChildInfo.js]: Please remove the negative margin and try to find an alternative approach.

File: js/screens/ChildInfo/ChildInfo.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { View, Text, TouchableOpacity, ScrollView } from 'react-native';
+import ProfileCard from '../../components/ProfileCard';
+import OutlinedButton from '../../components/OutlinedButton';
+import PropTypes from 'prop-types';
+import styles from './styles';
+import { colors } from '../../config/styles';
+import ChildInfoForm from '../../components/ChildInfoForm';
+
+const Button = ({ pressed }) => (
```
Comment (siddharthparmar7) [inline js/screens/ChildInfo/ChildInfo.js]: can't name it same as React-native component. Please rename it.

File: js/screens/ChildInfo/ChildInfo.js [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+  child,
+  editChildInfo,
+  showEditInfo,
+  updateChildInfo,
+  updateFormValues,
+}) => {
+  const { info } = child;
+  return (
+    <View style={styles.root}>
+      <ScrollView>
```
Comment (siddharthparmar7) [inline js/screens/ChildInfo/ChildInfo.js]: you can use your scroll view as root, don't need extra view component.

File: js/screens/ChildInfo/ChildInfoContainer.js [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+    };
+  }
+  _showEdit = () => {
+    this.setState({ editInfo: true });
+  };
+  _updateFormValues = values => {
+    this.setState({ formValues: values });
+  };
+  _updateChildInfo = () => {
+    // function to update database
```
Comment (siddharthparmar7) [inline js/screens/ChildInfo/ChildInfoContainer.js]: is this a TODO? if not then please remove it.

File: js/screens/ChildProfile/ChildProfile.js [hunk 9]
```diff
@@ -26,8 +26,11 @@ const ChildProfile = ({
           action={
             <OutlinedButton
               title="more info"
-              pressed={() => {}}
+              pressed={() => {
+                navigation.navigate('ChildInfo', { child: child });
+              }}
               color={colors.teal}
+              titleStyle={{ marginTop: -5, marginBottom: -7 }}
```
Comment (siddharthparmar7) [inline js/screens/ChildProfile/ChildProfile.js]: Please do not use negative margins. Find an alternative approach.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work! Please make the requested changes. Thank you.

File: js/components/ChildInfoForm/ChildInfoForm.js (hunk 2 above)
Comment (crathor) [reply js/components/ChildInfoForm/ChildInfoForm.js]: I removed validate, however even though I do not need onSubmit it is a required prop

File: js/screens/ChildProfile/ChildProfile.js (hunk 9 above)
Comment (crathor) [reply js/screens/ChildProfile/ChildProfile.js]: the only approach I could come up with is to not use the designer's font. the font comes with its own built-in padding. the margin removes it to make the button match the designer's vision.

File: js/screens/ChildInfo/ChildInfo.js (hunk 5 above)
Comment (crathor) [reply js/screens/ChildInfo/ChildInfo.js]: the only approach I could come up with is to not use the designer's font. the font comes with its own built-in padding. the margin removes it to make the button match the designer's vision.

**Review by siddharthparmar7 — APPROVED** [review]
Thank you!

**Pull request merged**